bin/logstash.lib.sh

```diff
@@ -112,7 +123,7 @@ setup_java() {
   fi
 
   if [ ! -x "$JAVACMD" ]; then
-    echo "could not find java; set JAVA_HOME or ensure java is in PATH"
+    echo "could not find java; set LS_JAVA_HOME or ensure java is in PATH"
```

Comment:
Should the first letter be upper cased? (It is a new sentence(. Also, I believe we should use proper punctuation, so should likely need a '.' at the end of all sentences.